tox.ini

```diff
@@ -36,6 +36,7 @@ deps=
   selenium==3.0.2
   pymongo==3.4.0
   toml<2.0.0
+  responses
```

Comment:
can we lock this down to the current milestone release? `responses<1.0` (since 0.9.0 is the current)

Reply:
OK